app/gen-server/entity/Group.ts

```diff
@@ -1,10 +1,14 @@
-import {BaseEntity, Column, Entity, JoinTable, ManyToMany, OneToOne, PrimaryGeneratedColumn} from "typeorm";
+import {BaseEntity, BeforeInsert, BeforeUpdate, Column, Entity, JoinTable, ManyToMany,
+  OneToOne, PrimaryGeneratedColumn} from "typeorm";
 
 import {AclRule} from "./AclRule";
 import {User} from "./User";
+import { ApiError } from "app/common/ApiError";
 
 @Entity({name: 'groups'})
 export class Group extends BaseEntity {
+  public static readonly ROLE_TYPE = 'role';
+  public static readonly RESOURCE_USERS_TYPE = 'resource users';
```

Comment:
I am not very fan of this name, I admit. I found the name `team` here and it's sounds more accurate and understandable :
https://github.com/gristlabs/grist-core/blob/7d0bbf3570f651bfe5c9484267a126a8b9218ee7/app/gen-server/lib/homedb/HomeDBManager.ts#L4195

Do you agree for this renaming?